drivers/node/vsphere/vsphere.go

```diff
@@ -0,0 +1,248 @@
+package vsphere
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/portworx/sched-ops/task"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/node/ssh"
+	"github.com/sirupsen/logrus"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+const (
+	// DriverName is the name of the vsphere driver
+	DriverName = "vsphere"
+	// Protocol is the protocol used
+	Protocol = "https://"
+)
+
+const (
+	vsphereUname = "VSPHERE_USER"
+	vspherePwd   = "VSPHERE_PWD"
+	vsphereIP    = "VSPHERE_HOST_IP"
+)
+
+const (
+	// DefaultUsername is the default username used for ssh operations
+	DefaultUsername = "root"
+	// VMReadyTimeout Timeout for checking VM power state
+	VMReadyTimeout = 3 * time.Minute
+	// VMRetryInterval interval for retry when checking VM power state
+	VMRetryInterval = 5 * time.Second
```

Comment:
rename it with the same name as above VMReadyRetryInterval

Reply:
Fixed.